cmd/mycelo/templates.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"math/big"
+	"math/rand"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/mycelo/config"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+type template interface {
+	createEnv(workdir string) (*config.Environment, error)
+}
+
+func templateFromString(templateStr string) template {
+	switch templateStr {
+	case "local":
+		return localEnv{}
+	case "testnet":
+		return testnetEnv{}
+	case "loadtest":
+		return loadtestEnv{}
+	}
+	return localEnv{}
+}
+
+type localEnv struct{}
+
+func (e localEnv) createEnv(workdir string) (*config.Environment, error) {
+	envCfg := &config.EnvConfig{
+		Mnemonic:           config.MustNewMnemonic(),
+		InitialValidators:  3,
+		ValidatorsPerGroup: 1,
+		DeveloperAccounts:  10,
+		ChainID:            big.NewInt(1000 * (1 + rand.Int63n(9999))),
+	}
+
+	env, err := config.NewEnv(workdir, envCfg)
```

Comment:
NewEnv generates the accounts, so if you modify envConfig later, you need to regenerate it. Tha's why it had the applyOverrides fn